src/renderer/views/Playlist/Playlist.js

```diff
@@ -420,6 +434,26 @@ export default defineComponent({
       showToast(this.$t('User Playlists.SinglePlaylistView.Toast.This playlist does not exist'))
     },
 
+    getDurationPlaylistItems: function () {
+      const modifiedPlaylistItems = deepCopy(this.playlistItems)
+      modifiedPlaylistItems.forEach(video => {
+        if (!videoDurationPresent(video)) {
+          const videoHistory = this.$store.getters.getHistoryCacheById[video.videoId]
+          if (typeof videoHistory !== 'undefined') video.lengthSeconds = videoHistory.lengthSeconds
+        }
+      })
+      return modifiedPlaylistItems
+    },
+
+    showNoticesSometimes: function (playlistItems) {
+      if (this.alreadyShownNotice) return
+      const anyVideoMissingDuration = playlistItems.some(v => !videoDurationPresent(v))
+      if (anyVideoMissingDuration) {
+        showToast(this.$t('User Playlists.SinglePlaylistView.Toast.This playlist has a video with a duration error'), 5000)
+        this.alreadyShownNotice = true
+      }
+    },
```

Comment:
Loop once only for performance?
Method name would be too long if I add "sometimes with notice" :P
```suggestion
    getPlaylistItemsWithDuration() {
      const modifiedPlaylistItems = deepCopy(this.playlistItems)
      let anyVideoMissingDuration = false
      modifiedPlaylistItems.forEach(video => {
        if (videoDurationPresent(video)) { return }

        const videoHistory = this.$store.getters.getHistoryCacheById[video.videoId]
        if (typeof videoHistory !== 'undefined') {
          video.lengthSeconds = videoHistory.lengthSeconds
        } else {
          // Mark at least one video have no duration, show notice later
          // Also assign fallback duration here
          anyVideoMissingDuration = true
          video.lengthSeconds = 0
        }
      })
      
      // Show notice if not already shown before returning playlist items
      if (anyVideoMissingDuration && !this.alreadyShownNotice) {
        showToast(this.$t('User Playlists.SinglePlaylistView.Toast.This playlist has a video with a duration error'), 5000)
        this.alreadyShownNotice = true
      }
      
      return modifiedPlaylistItems
    },
```